package.json

```diff
@@ -1,35 +1,42 @@
 {
-  "name": "javascript-action",
+  "name": "check-licenses",
   "version": "1.0.0",
-  "description": "JavaScript Action Template",
+  "description": "A Github action that detects new dependencies in PRs and adds reminder comments to review licensing details",
   "main": "index.js",
   "scripts": {
+    "clean": "rm -rf dist",
     "lint": "eslint .",
-    "prepare": "ncc build index.js -o dist --source-map --license licenses.txt",
-    "test": "jest",
-    "all": "npm run lint && npm run prepare && npm run test"
+    "build": "ncc build src/index.js -o dist --source-map --license licenses.txt",
+    "test": "jest"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/actions/javascript-action.git"
+    "url": "git+https://github.com/oncai/check-licenses.git"
   },
   "keywords": [
     "GitHub",
     "Actions",
-    "JavaScript"
+    "Dependency",
+    "License"
   ],
   "author": "",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/actions/javascript-action/issues"
+    "url": "https://github.com/oncai/check-licenses/issues"
   },
-  "homepage": "https://github.com/actions/javascript-action#readme",
+  "homepage": "https://github.com/oncai/check-licenses#readme",
   "dependencies": {
-    "@actions/core": "^1.10.0"
+    "@actions/core": "^1.10.0",
+    "@octokit/action": "^4.0.10",
+    "dotenv": "^16.0.3",
+    "mustache": "^4.2.0",
+    "octokit": "^2.0.10",
+    "parse-diff": "^0.10.0"
   },
   "devDependencies": {
     "@vercel/ncc": "^0.31.1",
     "eslint": "^8.0.0",
-    "jest": "^27.2.5"
+    "jest": "^27.2.5",
+    "run-all": "^1.0.1"
```

Comment:
New package: **run-all**

version: **^1.0.1**
license: **https:&#x2F;&#x2F;github.com&#x2F;mysticatea&#x2F;run-all**
homepage: 

Please review Onc AI's [Open Source Licenses and Processes](https://onc-ai.atlassian.net/wiki/spaces/RD/pages/1740570625/Open+Source+Licenses+and+Processes) and consult @oncai/open-source-compliance as needed.